components/custom_status/custom_status.scss

```diff
@@ -0,0 +1,181 @@
+.StatusModal {
+    width: 600px !important;
```

Comment:
Any way to avoid `!important` here?

Reply:
This is to override the styles of another class. Added this comment in the code to explain.